assets/js/blocks/cart/inner-blocks/proceed-to-checkout-block/block.tsx

```diff
@@ -55,12 +55,21 @@ const Block = ( {
 			global.removeEventListener( 'pageshow', hideSpinner );
 		};
 	}, [] );
-
+	const { getValidationError } = useSelect( ( select ) => {
+		return {
+			getValidationError: select(
+				'wc/store/validation'
+			).getValidationError(),
+		};
+	} );
+	const hasValidationError = !! getValidationError(
+		'wc/cart/address-update'
+	);
 	const submitContainerContents = (
 		<Button
 			className="wc-block-cart__submit-button"
 			href={ link || CHECKOUT_URL }
-			disabled={ isCalculating }
+			disabled={ isCalculating || hasValidationError }
```

Comment:
Are we sure we want to block the user from proceeding to checkout if they made an error in the shipping calculator? I think this adds unnecessary friction to the shopper experience, so we should probably be fine with displaying the error, but letting them proceed.

Reply:
I agree, this is only an estimation, not a final step.